tests/unittests/tests-crypto/tests-crypto-modes-ccm.c

```diff
@@ -121,38 +140,55 @@ static void test_decrypt_op(uint8_t* key, uint8_t key_len, uint8_t* adata,
 
 }
 
+#define do_test_encrypt_op(name) do { \
```

Comment:
not a big fan of preprocessor macros, but I guess it's fine in this case.

Reply:
Its to prepare for having 28 tests and the macro is local only.

Reply:
while I would agree with @aabadie for production code, IMHO its fine for testing and I would leave as proposed by @cladmi here.